packages/datadog-instrumentations/src/jest.js

```diff
@@ -451,7 +474,18 @@ addHook({
       return getTestPaths.apply(this, arguments)
     }
 
-    const [{ rootDir }] = arguments
+    const [{ rootDir, shard }] = arguments
+
+    if (shard && shard.shardIndex) {
```

Comment:
2 questions in this line...
1. `skippableSuites` are not going to be setted as `[]` again if the test are running in "shard" mode. Is this expected?
2. `shardIndex` looks like something that start from `0`, so if `shard.shardIndex === 0` this condition will be `false`
  2.1. So, point 1 is resolved, because `skippableSuites` are going to be set as `[]` when it is 0
  2.1. But, We will execute `getJestSuitesToRun` one more time than expected, in `shard` method and also few lines below.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/3452/commits/cdf4460dcbb87ec370ee52ff90c9303293d24768